scripts/create-vault.sh

```
@@ -59,20 +80,80 @@ else
+$TX_SCRIPT gov submit-proposal "$PROPOSAL_FILE" --from "$ADMIN"
 if [ $? -ne 0 ]; then
-  echo "ERROR: Failed to create vault."
+  echo "ERROR: Failed to submit the vault creation proposal."
   exit 1
 fi
 
-echo "Vault creation command sent successfully for $UNDERLYING_ASSET."
+# --- 4. Vote on the proposal ---
+# Identify the proposal by the message it carries rather than by taking the newest
+# proposal, so an unrelated proposal submitted concurrently is never picked up.
+PROPOSAL_ID=$($SIMD_BIN query gov proposals $HOME_DIR --output json 2>/dev/null | jq -r --arg denom "$SHARE_DENOM" '
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)